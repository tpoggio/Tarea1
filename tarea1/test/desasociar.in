asociar 15 4.0
desasociar 15
esClave 15
asociar 15 4.0
esClave 15
valor 15
asociar 95 24.0
valor 95
asociar 85 2.0
asociar 65 3.0
asociar 25 67.0
asociar 35 -12.0
asociar 75 2.5
asociar 45 2.1
asociar 55 -24.0
desasociar 85
asociar 85 32.0
valor 85
asociar 5 10.0
valor 5
asociar 1 10.0
esClave 1
desasociar 35
esClave 35
asociar 1 10.0
esClave 1
valor 1
Fin