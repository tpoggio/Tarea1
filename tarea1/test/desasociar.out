1>Asociar.
2>Desasociar. 
3>15 no es clave. 
4>Asociar.
5>15 es clave. 
6>15 está asociado a 4.00. 
7>Asociar.
8>95 está asociado a 24.00. 
9>Asociar.
10>Asociar.
11>Asociar.
12>Asociar.
13>Asociar.
14>Asociar.
15>Asociar.
16>Desasociar. 
17>Asociar.
18>85 está asociado a 32.00. 
19>Asociar.
20>5 está asociado a 10.00. 
21>Asociar.
22>1 no es clave. 
23>Desasociar. 
24>35 no es clave. 
25>Asociar.
26>1 es clave. 
27>1 está asociado a 10.00. 
28>Fin.
